### ISSUE TITLE
image upload from disk

### ISSUE THREAD

**Issue by izielin** [pylixm/django-mdeditor#48]
There is some way to insert image directly from disk or only from http address?

**Comment by pylixm** [comment]
Yes, support uploading pictures from hard disk .Images in http or https can be written directly in the document.

**Comment by izielin** [comment]
So, why when I try upload some img from disk I got notification that "address can't be blank" or sth like this?

**Comment by pylixm** [comment]
Any ideas?Can a minimum reproducible sample be provided?

**Comment by izielin** [comment]
I don't know if it could help but here is project where I use mdeditor (app blog). Maybe I forget to add some important code line but I think that everything is correct.
https://github.com/izielin/ecg_vot_pl
I would be grateful if you could help me.

**Comment by pylixm** [comment]
No response for a long time, temporarily closed. If you have any questions, you can open another issue.
